### DISCUSSION TITLE
question: bootstrap DNS strategy

### DISCUSSION THREAD

**Discussion by VirgilSone** [AdguardTeam/AdGuardHome#3546] [Q&A]
I understand there are several strategy for upstream dns (parallel, fastest, etc.), but what is the strategy for bootstrap DNS?
Do they share the same way?
For example if I select "parallel quires" as upstream dns strategy and set several different bootstrap DNS, will this setting also appy to bootstrap DNS?

**Comment by ameshkov** [comment]
Bootstrap DNS are similar, always "parallel" query.
